bin/pycbc_compress_bank

```diff
@@ -125,8 +129,8 @@ else:
 logging.info("getting needed dfs")
 # we'll ensure that the there are atleast 2 samples in a waveform
 seg_lens = 4*numpy.array([max(4./args.sample_rate,
-    2**numpy.ceil(numpy.log2(compress.rough_time_estimate(m1, m2, fmin))))
-    for m1,m2 in zip(templates.mass1, templates.mass2)])
+    2**numpy.ceil(numpy.log2(compress.rough_time_estimate(m1, m2, flow))))
+    for m1,m2,flow in zip(templates.mass1, templates.mass2, templates.f_lower)])
```

Comment:
Can we not just require f_lower in the input template banks? The HDF banks (whether using variable f-lower or not) will always contain this column now.